From 00ef927f51859ed8600dcc3ac8bfa45dc57be06e Mon Sep 17 00:00:00 2001
From: Pete Zaitcev <zaitcev@kotori.zaitcev.us>
Date: Thu, 18 Sep 2014 19:25:22 -0600
Subject: [PATCH] Patch docs to use oslo.sphinx

Patching the source this way is the technique that most our projects
use for now. Some even use sed(1)! Ask Padraig as to why this is.
Sooner or later we need to migrate to oslosphinx 2.0, but not today.
---
 doc/source/conf.py    | 2 +-
 test-requirements.txt | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index 0b3e7e1..bfb1d76 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -32,7 +32,7 @@ sys.path.insert(0, ROOT)
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = ['sphinx.ext.autodoc', 'sphinx.ext.doctest', 'sphinx.ext.todo',
-              'sphinx.ext.coverage', 'oslosphinx']
+              'sphinx.ext.coverage', 'oslo.sphinx']
 
 autoclass_content = 'both'
 autodoc_default_flags = ['members', 'undoc-members', 'show-inheritance']
diff --git a/test-requirements.txt b/test-requirements.txt
index 5427920..168cede 100644
--- a/test-requirements.txt
+++ b/test-requirements.txt
@@ -3,7 +3,7 @@ hacking>=0.8.0,<0.9
 coverage>=3.6
 discover
 mock>=1.0
-oslosphinx
+oslo.sphinx
 python-keystoneclient>=0.7.0
 sphinx>=1.1.2,<1.2
 testrepository>=0.0.18
